analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/ClearTextProtocolsAreSensitive.cs

```diff
@@ -148,14 +148,66 @@ public void Ftp()
 
         private readonly List<string> links = new List<string>
         {
-            "http://example.com", // Noncompliant
-            "Http://example.com", // Noncompliant
-            "HTTP://example.com", // Noncompliant
-            "https://example.com",
-            "telnet://anonymous@example.com", // Noncompliant
-            "TELNET://anonymous@example.com", // Noncompliant
-            "ftp://anonymous@example.com", // Noncompliant
-            "FTP://anonymous@example.com" // Noncompliant
+            "http://foo.com", // Noncompliant
+            "Http://foo.com", // Noncompliant
+            "HTTP://foo.com", // Noncompliant
+            "https://foo.com",
+            "telnet://anonymous@foo.com", // Noncompliant
+            "TELNET://anonymous@foo.com", // Noncompliant
+            "ftp://anonymous@foo.com", // Noncompliant
+            "FTP://anonymous@foo.com" // Noncompliant
+        };
+
+        private readonly List<string> commonlyUsedXmlDomains = new List<string>
+        {
+            "http://www.w3.org",
+            "http://xml.apache.org",
+            "http://schemas.xmlsoap.org",
+            "http://schemas.openxmlformats.org",
+            "http://rdfs.org",
+            "http://purl.org",
+            "http://xmlns.com",
+            "http://schemas.google.com",
+            "http://a9.com",
+            "http://ns.adobe.com",
+            "http://ltsc.ieee.org",
+            "http://docbook.org",
+            "http://graphml.graphdrawing.org",
+            "http://json-schema.org",
+            "http://email:password@subdomain.www.w3.org", // Noncompliant
+            "http://domain.com/.www.w3.org", // Noncompliant
+            "http://domain.com/path?domain=.www.w3.org", // Noncompliant
+            "http://domain.com?q=.www.w3.org", // Noncompliant
+            "http://domain.com#.www.w3.org", // Noncompliant
+            "http://subdomain.www.w3.org", // Noncompliant
+            "http://subdomain.xml.apache.org", // Noncompliant
+            "http://subdomain.schemas.xmlsoap.org", // Noncompliant
+            "http://subdomain.schemas.openxmlformats.org", // Noncompliant
+            "http://subdomain.rdfs.org", // Noncompliant
+            "http://subdomain.purl.org", // Noncompliant
+            "http://subdomain.xmlns.com", // Noncompliant
+            "http://subdomain.schemas.google.com", // Noncompliant
+            "http://subdomain.a9.com", // Noncompliant
+            "http://subdomain.ns.adobe.com", // Noncompliant
+            "http://subdomain.ltsc.ieee.org", // Noncompliant
+            "http://subdomain.docbook.org", // Noncompliant
+            "http://subdomain.graphml.graphdrawing.org", // Noncompliant
+            "http://subdomain.json-schema.org", // Noncompliant
```

Comment:
Indenting the annotations, for example like this, makes it easier to read.
```suggestion
            "http://subdomain.rdfs.org", 		// Noncompliant
            "http://subdomain.purl.org", 		// Noncompliant
            "http://subdomain.xmlns.com", 		// Noncompliant
            "http://subdomain.schemas.google.com", // Noncompliant
            "http://subdomain.a9.com", 			// Noncompliant
            "http://subdomain.ns.adobe.com", 	// Noncompliant
            "http://subdomain.ltsc.ieee.org", 	// Noncompliant
            "http://subdomain.docbook.org", 	// Noncompliant
            "http://subdomain.graphml.graphdrawing.org", // Noncompliant
            "http://subdomain.json-schema.org", // Noncompliant
```
We don't need to ident all of them to far end, but indenting them in a reasonable groups helps a lot to separate testcase from it's expected result without optical mix of action (comment) and data (syntax) on nearby rows.